R/github-actions.R

```diff
@@ -4,51 +4,44 @@
 #' GitHub using [GitHub Actions](https://github.com/features/actions). These functions
 #' - Add the necessary configuration files and place them in `.Rbuildignore`.
 #' - Provide the markdown to insert a badge into your README
-#' @name actions
+#' @name github_actions
 NULL
 
 #' @section `use_github_actions()`:
-#' Adds a basic ‘R.yaml’ file to the `.github/workflows` directory of a
+#' Adds a basic ‘R-CMD-check.yaml’ file to the `.github/workflows` directory of a
 #'  package. This is a configuration file for the [GitHub
 #'  Actions](https://github.com/features/actions) service.
-#' @rdname actions
+#' @rdname github_actions
 #' @export
 use_github_actions <- function() {
   check_uses_github()
 
-  new <- use_action_check_release()
-
-  if (!new) {
-    return(invisible(new))
-  }
-  use_github_actions_badge("R-CMD-check")
-
-  invisible(new)
+  use_github_action_check_release()
 }
 
-#' @section `use_tidy_actions()`:
+#' @section `use_github_actions_tidy()`:
 #' Sets up tidyverse actions that check the R versions officially supported by
 #'   the tidyverse, (current release, devel and four previous versions). It also
 #'   adds two commands to be used in pull requests, `\document` to run
 #'   `roxygen2::roxygenise()` and update the PR, and `\style` to run
 #'   `styler::style_pkg()` and update the PR.
-#' @rdname actions
+#' @rdname github_actions
 #' @export
-use_tidy_actions <- function() {
+use_github_actions_tidy <- function() {
   check_uses_github()
 
-  new <- use_action_check_full() && use_action_pr_commands()
+  new <- use_github_action_check_full() && use_github_action_pr_commands()
```

Comment:
One issue with this is that if a package does _not_ have a README, then `use_github_action_check_full()` will return `FALSE`, so the pr commands won't be added.

I guess I decided this is rare enough not to worry about.